qa/tasks/rgw.py

```diff
@@ -1256,6 +1179,7 @@ def task(ctx, config):
         lambda: create_nonregion_pools(
             ctx=ctx, config=config, regions=regions),
         ]
+    log.debug('Nonregion pools created')
```

Comment:
I think it's fine because it's aligned with subtasks above it and multisite right below it?